### ISSUE TITLE
Config.json should not have non-web actions

### ISSUE THREAD

**Issue by PulkitChadhaAdobe** [adobe/aio-cli-plugin-app#182]
Config.json should not have non-web actions

I have a dew non-web actions for authentication and accessToken generation that I don’t want to enable on web. I can do that through the annotation on the manifest file. However, the issue is aio app deploy does not take this into account and lists these actions in the config.json file with a derived URL to invoke the action, which of course, results in the component rendering the invoke button which when clicked results in error.

**Comment by moritzraho** [comment]
@PulkitChadhaAdobe thanks for raising this issue, indeed I think that non-web actions should not be invoked from the UI, meaning that we should probably not show those urls in the config.json.

I will mark this as a bug. thanks!

> I have a dew non-web actions for authentication and accessToken generation
out of curiosity, can't you rely on the user token generated by the Experience cloud shell (available in your ims pros in App.js)

**Comment by PulkitChadhaAdobe** [comment]
@moritzraho , turns out my use-case was redundant. I wanted to migrate my legacy JWT token and accessToken actions into the app. It turns out I can leverage the shared token and no longer needed.

**Comment by meryllblanchet** [comment]
Thanks for the feedback @PulkitChadhaAdobe . I'll then close the issue.

**Comment by moritzraho** [comment]
@PulkitChadhaAdobe thanks for the follow up, good that you are unblocked !
